test/CETEIcean.css

```diff
@@ -897,20 +898,3 @@ tei-wit {
 tei-witdetail {
   display:none;
 }
-
-/* styles for HTML shell and HTML elements in TEI (e.g.,<a> and <img>) */
-html > body {
-  margin: 0;
-  padding: 0;
-}
-html {
-  margin: 0;
-  padding: 0;
-  margin-left: 2em;
-  margin-right: 4em;
-  padding: 2.5em;
-  font-family: "Lucida Grande", "Cardo", "Arial Unicode MS","Galilee Unicode Gk", "New Athena Unicode", "Athena Unicode", "Palatino Linotype", "Titus Cyberbit Basic", "Vusillus Old Face", "Alphabetum", "Galatia SIL", "Code 2000", "GentiumAlt", "Gentium", "Minion Pro", "GeorgiaGreek", "Vusillus Old Face Italic", "Everson Mono", "Aristarcoj", "Porson", "Legendum", "Aisa Unicode", "Hindsight Unicode", "Caslon", "Verdana", "Tahoma";
-  font-size: 12pt;
-  background-color: white;
-  color: #292929;
-}
```

Comment:
I know (from experience) that having this is frustrating, but this CSS file is mostly meant as an example style that works out of the box. I would slightly prefer to keep it here so that a CETEI rendering with default settings looks good and have advanced users either modify this file for themselves or override in a separate CSS.

Reply:
Or to work with your solution of splitting this into two CSS files: we could make CETEIcean.css include html.css and tei.css (replaces CETEIcean.css). @hcayless thoughts?